src/math/exp_fcn.c

```diff
@@ -0,0 +1,171 @@
+// SPDX-License-Identifier: BSD-3-Clause
+//
+// Copyright(c) 2022 Intel Corporation. All rights reserved.
+//
+// Author: Shriram Shastry <malladi.sastry@linux.intel.com>
+//
+//
+
+/* Include Files */
+#include <sof/math/exp_fcn.h>
+#include <sof/math/numbers.h>
+#include <stdint.h>
+#include <stdbool.h>
+
+/* Function Declarations */
+static inline int64_t lomul_s64_SR_sat_near(int64_t a, int64_t b);
```

Comment:
you shouldn't need to forward declare statics, please re-order these so this is unneeded

Reply:
Thank you!
Would you kindly provide further information about the adverse effects and other problems it may cause?

Reply:
@ShriramShastry to clarify any functions that use this one should come after it in the file so you can delete this line

Reply:
Sure, Thanks!!